src/geowombat/moving/meson.build

```diff
@@ -0,0 +1,11 @@
+numpy_dep = dependency('numpy', version: '>=2.0.0',required: true)
+omp = dependency('openmp')
+
+import('python').find_installation().extension_module(
```

Comment:
The `py_mod` and `py3` variables established in the root meson.build file are available for use here as well.

While you are at it, you may choose to do dependency lookups (`dependency()`) a single time in the root meson.build file as well, rather than once per subdir using it. The lookups are cached either way so it won't re-run the finders (numpy-config etc.), but it will emit additional log lines.